objects/src/mock/chain.rs

```diff
@@ -463,6 +525,11 @@ mod mock {
             Ok(())
         }
 
+        /// Add a known [Account] to the mock chain.
+        pub fn add_account(&mut self, account: Account, seed: Word) {
+            self.pending_objects.accounts.push((account, seed));
+        }
```

Comment:
Should there be a check here (e.g., via an `assert!`) to make sure we don't add an account the same ID twice?

Reply:
good point, added.